src/lib/libwasmfs.js

```diff
@@ -111,7 +111,11 @@ FS.init();
         if (!wasmFSPreloadingFlushed) {
           wasmFSPreloadedDirs.push({parentPath: parent, childName: part});
         } else {
-          FS.mkdir(current);
+          try {
+            FS.mkdir(current);
+          } catch (e) {
+            if (e.errno != {{{ cDefs.EEXIST }}}) throw e;
```

Comment:
I looks like the code in `libjs.js` is even more relaxed than this: 

```
        try {                                                                         
          FS.mkdir(current);                                                          
        } catch (e) {                                                                 
          // ignore EEXIST                                                            
        }   
```

I guess they should at least be consistent?

Reply:
I feel like it intended to specifically ignore EEXIST, but was written in a buggy way; it’s plausible existing code relies on that permissiveness so I hesitated to change it.

Reply:
I think we we should change libfs.js too.  If the tests pass that should be good signal that its good.   I don't think there are very many users of this API outside of the file packager.

Reply:
Done!